packages/core/scripts/serverless-orchestration/ServerlessHub.js

```diff
@@ -99,18 +99,27 @@ hub.post("/", async (req, res) => {
     let errorOutputs = {};
     let validOutputs = {};
     results.forEach((result, index) => {
-      if (result.status == "rejected" || result?.value?.execResponse?.error || result?.reason?.code == "500") {
-        // If the child process in the spoke crashed it will return 500 (rejected). OR If the child process exited
-        // correctly but contained an error.
+      // If the child process in the spoke crashed it will return 500 (rejected). OR If the child process exited
```

Comment:
this is the main refactor to roll back to supporting LTS JS (removing the `?.` syntax)